docs/getting-started/products/ylockers/overview.md

```diff
@@ -1,8 +1,111 @@
 # Overview
-yLockers is a category of assets built by Yearn designed to tokenize locked governance positions in external DeFi protocols. This asset type is often called a "Liquid Locker".
 
-To achieve this, Yearn has deployed a system of smart contracts that allows users to permissionlessly max-lock their governance tokens to a central Yearn in exchange for a yLocker token (e.g., yCRV, yPRISMA, etc.) at a rate of 1:1. 
+:::info
+
+The new Ylockers ecosystem is rolling out on May 9th, 2024 with the launch of the *YearnBoostedStaker* (YBS) contract and a new V3 Auto-Compounder vault. These new contracts will be available for yPRISMA initially and then be rolled out for yCRV a month later on June 6th. Before June 6th, yCRV will continue to operate with the existing st-yCRV and lp-yCRV V2 vaults.
+
+:::
+
+yLockers are a category of assets built by Yearn designed to tokenize locked governance positions in external DeFi protocols. This asset type is often called a "Liquid Locker".
+
+To achieve this, Yearn has deployed a system of smart contracts that allows users to permissionlessly max-lock their governance tokens to a central Yearn in exchange for a yLocker token (e.g., yCRV, yPRISMA, etc.) at a rate of 1:1.
 
 The benefit of doing this is to provide the end user with a fully transferrable and liquid token that can still receive a share of governance benefits like yield or voting power depending on the protocol. No longer is there a need to lock up your asset value for up to 4 full years!
 
 > **⚠️ Important**: Liquid locker tokens are not redeemable for the underlying locked tokens. This is not possible. But because they are liquid, they can be traded on decentralized exchanges, and bought and sold at market value.
+
+## yLocker Products
+
+* [yPrisma](../ylockers/yprisma/overview)
+* [yCRV](../ylockers/ycrv/overview)
+
+## Earning Yield with yLockers
+
+Within the yLockers ecosystem, there are two distinct ways to earn yield:
+
+1. Deposit your yLocker tokens into YBS (earn stablecoins)
+2. Deposit your yLocker tokens into the V3 Auto-Compounder (earn yLocker tokens)
```

Comment:
Nit: this is will technically not be a V3 vault for yCRV...so maybe best to just call it `Auto-Compounder Vault`

Reply:
fixed